example/points/Makefile.am

```diff
@@ -7,12 +7,19 @@ if P4EST_ENABLE_BUILD_2D
 bin_PROGRAMS += example/points/p4est_points
 example_points_p4est_points_SOURCES = example/points/points2.c
 
+bin_PROGRAMS += example/points/p4est_generate_points
+example_points_p4est_generate_points_SOURCES = example/points/generate_points.c
+
 LINT_CSOURCES += $(example_points_p4est_points_SOURCES)
 endif
 
 if P4EST_ENABLE_BUILD_3D
 bin_PROGRAMS += example/points/p8est_points
 example_points_p8est_points_SOURCES = example/points/points3.c
 
+bin_PROGRAMS += example/points/p8est_generate_points
+example_points_p8est_generate_points_SOURCES = example/points/generate_points.c
+example_points_p8est_generate_points_CPPFLAGS = -DTHREED $(AM_CPPFLAGS)
```

Comment:
See below comment; such a THREED macro is usually not necessary in how we write dimension-dependent code.